pagopa/io-app
# feat(IT Wallet): [SIW-221] Add home screen scaffold

## Depends on #4745 

## Short description
This PR adds the home screen scaffold with the top categories bar and the activation banner.

## List of changes proposed in this pull request
- `locales/en/index.yml` & `locales/it/index.yml`: Adds activation banner and categories copy;
- `ts/components/messages/MessagesInbox.tsx`: Adds missing prop for the new `ItwActionBanner` component (see below for its changes);
- `ts/features/it-wallet/components/ItwActionBanner.tsx`: Adds props for customizing the activation banner component; 
- `ts/features/it-wallet/components/design/BadgeButton.tsx`: Creates a new component which consists of a pressable `Badge`;
- `ts/features/it-wallet/screens/ItwHomeScreen.tsx`: Adds the home screen component;
- `ts/navigation/TabNavigator.tsx`: Updates the route with the new screen component.

## How to test
Start the app with the FF enabled and open the `Wallet` section. Test the banner and also the top categories. Only the selected category should have a grey background.
